about.md

```diff
@@ -25,7 +25,7 @@ Programmers, engineers, and scientists with an interest in any aspect of music.
 
 ## What can I make?
 
-Wiktionary gives one definition of hack as “To work on an intimately technical level”  (http://en.wiktionary.org/wiki/hack).  An appropriate hack at this event would work on an intimately technical level with any subject surrounding music, such as making sound, making instruments, making compositions, analysis of a piece of music, analysis of a music collection, analysis of music listening data, making music presentation platforms, making music sharing apps, making music sales apps, etc, etc, etc…
+Wiktionary gives one definition of [hack](http://en.wiktionary.org/wiki/hack) as “To work with on an intimately technical level.”  An appropriate hack at this event would work on an intimately technical level with any subject surrounding music, such as making sound, making instruments, making compositions, analysis of a piece of music, analysis of a music collection, analysis of music listening data, making music presentation platforms, making music sharing apps, making music sales apps, etc, etc, etc…
```

Comment:
1. I thought I had updated the about to the master branch before making revisions. Was I wrong? I thought the revisions were being made on top of what was on the current mmhpreview.com master about.md
2. I made a commit mistake. I checked out the about branch, and somehow switched to my local master and committed my changes to that, then made a pull request on the about branch, which hadn't undergone changes. 
I fixed it by
- reverting master to the previous commit, then pushing
- copying my prior changes to the master's about.md to the about branch's about.md
- commiting and pushing the new about.
3. There are more changes. I don't know if you agree with all of them. That was one of them. The rest have to do with the time frame, how we display links, etc.